tests/e2e/vLLM/recipes/INT8/recipe_int8_channel_weight_dynamic_per_token.yaml

```diff
@@ -3,7 +3,7 @@ quant_stage:
     SmoothQuantModifier:
       smoothing_strength: 0.8
     GPTQModifier:
-      ignore: [lm_head]
+      ignore: ["lm_head", "re:vision_tower.*", "re:multi_modal_projector.*", "re:visual.*", "re:vision_model.*"]
```

Comment:
dynamic per token is fast to process to completion. Why are we ignoring these now?

Reply:
we want to avoid these layers when quantizing, as it affects model performance (as in accuracy, as opposed to runtime considerations)